doc/api/errors.md

```diff
@@ -214,6 +214,11 @@ A MediaError can have the following codes (``code`` property):
     contain any playable Representation. This usually happen when every
     Representation has been blacklisted due to encryption limitations.
 
+  - ``"MANIFEST_UPDATE_ERROR"``: This error should never be emitted, it is a
```

Comment:
Is it really necesseraly a RxPlayer bug if it happens ? It may be a server-side issue, no ?
I wouldn't be so sure in our documentation.

Reply:
No, it should never be emitted as it is normally directly catched by the RxPlayer which will perform a full Manifest update instead